specs/StatusBarMessage.md

```diff
@@ -0,0 +1,106 @@
+<!-- USAGE
+  * Fill in each of the sections (like Background) below
+  * Wrap code with `single line of code` or ```code block```
+  * Before submitting, delete all <!-- TEMPLATE marked comments in this file,
+    and the following quote banner:
+-->
+# Background
+The browser has a Status bar that displays text when hovering over a link, or performing some activity. Currently, 
+developers are able to opt in to disable showing the status bar through the browser 
+settings.
+
+Developers would also like to be able to opt in to intercept the messages which would 
+normally be displayed by the Status bar, and show it using thier own custom UI. 
```

Comment:
Presumably if I disable the status bar, the WebView still raises `StatusBarMessageChanged` events so I can show them in my alternate status bar. (In other words, disabling the status bar does not disable status bar messages, although it does mean that you cannot see those messages by default.)